backend/api/groups.py

```diff
@@ -86,3 +86,50 @@ async def get_groups(
                      for group in groups
                      if user.id in group["member_ids"]]
     return group_objects
+
+
+@router.put("/{group_id}", status_code=status.HTTP_200_OK,
+            response_model=models.Group)
+async def update_group(
+        group_id: PydanticObjectId,
+        request: models.Group,
```

Comment:
This should be an object of type UpdateGroupRequest (a newly defined class where you specify just the required fields for the update - e.g., for now, the group_id field is present both as a query parameter and in the request body)

Reply:
Please update also in the response_model field of the decorator